core/utils/isolation-manager.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Elementor\Core\Utils;
+
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Isolation_Manager {
+	/**
+	 * @param string $adapter_name
+	 */
+	public static function get_adapter( string $adapter_name ) {
+		$container = PLugin::$instance->elementor_container();
+		return $container->get( $adapter_name );
```

Comment:
I would wrap it with `try/catch` to avoid errors, and return null as a fallback. WDYT?

Reply:
This can also be an abstract class, could't it?